### ISSUE TITLE
[Bug] Main Site Auction Index references incorrect auction end time

### ISSUE THREAD

**Issue by clatwell** [Good-Bids/goodbids#910]
### TL;DR 

If an Auction's Close Date/Time has been updated due to a Bid Extension, the Auction Grid card on the Main Site (Auction Index, My Account / Dashboard) does not update to reference the new Close Date/Time. Instead, it displays the scheduled Auction End Time. The Auction Grid card on the NP Site (Auction Index, My Account / Dashboard) and the Auction page display the correct Close Date/Time. 

### Steps to Reproduce 

- On Staging, [create a new Auction](https://staging.goodbids.org/seagull/wp-admin/edit.php?post_type=gb-auction) on the Seagull Nonprofit Site with these times: 
 - Start Time: 5 min from now 
 - End Time: 1h from now 
 - Bid Extension: 50 min 
- Publish the Auction 
- When it goes live, confirm that the end time is correct on the [Main Site Auction Index](https://staging.goodbids.org/explore-auctions/), [NP Auction Index](https://staging.goodbids.org/seagull/explore-auctions/), and Auction page. 
- 45 minutes before the Auction is scheduled to end, place a bid. This will extend the Close Date Time by 5 minutes. 
- You'll now see a discrepancy between the end time on the Main Site pages and the Nonprofit Site pages: 
  - Main Site Auction Index: Shows scheduled end time 
  - Main Site / My Account Dashboard: Shows scheduled end time 
  - NP Site Auction Index: Shows actual close time 
  - NP Site Auction Page: Shows actual close time 
  - NP Site / My Account Dashboard: Shows actual close time 

You can also check out [this auction](https://staging.goodbids.org/seagull/auction/bid-extension-time-test/) on Staging: 

- [Main Site Auction Index](https://staging.goodbids.org/explore-auctions/): Closing at 1pm (incorrect)
- [NP Site Auction Index](https://staging.goodbids.org/seagull/explore-auctions/): Closing at 1:03pm 
- [Auction Page](https://staging.goodbids.org/seagull/auction/bid-extension-time-test/): Closing at 1:03pm
